Graphics/Implicit/ObjectUtil/GetImplicit3.hs

```diff
@@ -38,61 +36,11 @@ getImplicit3 (Cylinder h r1 r2) = \(x,y,z) ->
         θ = atan2 (r2-r1) h
     in
         max (d * cos θ) (abs (z-h/2) - (h/2))
--- (Rounded) CSG
-getImplicit3 (Complement3 symbObj) =
-    let
-        obj = getImplicit3 symbObj
-    in
-        \p -> - obj p
-getImplicit3 (UnionR3 r symbObjs) =
-  \p -> rminimum r $ fmap ($p) $ getImplicit3 <$> symbObjs
-
-getImplicit3 (IntersectR3 r symbObjs) =
-  \p -> rmaximum r $ fmap ($p) $ getImplicit3 <$> symbObjs
-
-getImplicit3 (DifferenceR3 r symbObj symbObjs) =
-    let
-        tailObjs = getImplicit3 <$> symbObjs
-        headObj = getImplicit3 symbObj
-        complement :: Obj3 -> ℝ3 -> ℝ
-        complement obj' p = - obj' p
-    in
-      \p -> do
-        let
-          maxTail = rmaximum r $ fmap ($p) $ complement <$> tailObjs
-        if maxTail > -minℝ && maxTail < minℝ
-          then rmax r (headObj p) minℝ
-          else rmax r (headObj p) maxTail
-
 -- Simple transforms
-getImplicit3 (Translate3 v symbObj) =
-    let
-        obj = getImplicit3 symbObj
-    in
-        \p -> obj (p ^-^ v)
-getImplicit3 (Scale3 s@(sx,sy,sz) symbObj) =
-    let
-        obj = getImplicit3 symbObj
-        k = abs (sx*sy*sz) ** (1/3)
```

Comment:
@julialongtin do you know what this `k` thing does? I'm not sure how to generalize this operation between 2d and 3d.

Reply:
looks like normalization in 3d, to me.

Reply:
for the 2d case, that is sqrt (abs( sx*sy)). for 3d, it's cubic_root (abs(sx*sy*sz))

Reply:
> `sqrt (abs( sx * sy))`. for 3d, it's `cubic_root (abs(sx*sy*sz))`

fixing formatting for my own purposes